sdk/containerregistry/azure-containerregistry/tests/test_container_registry_client.py

```diff
@@ -603,9 +604,50 @@ def test_get_misspell_property(self, containerregistry_endpoint):
         assert last_udpated_on == last_updated_on
     
     @acr_preparer()
-    def test_upload_manifest(self, containerregistry_endpoint):
+    def test_upload_oci_manifest(self, containerregistry_endpoint):
         repo = self.get_resource_name("repo")
         manifest = self.create_oci_manifest()
         client = self.create_registry_client(containerregistry_endpoint)
         
         client.upload_manifest(repo, manifest)
+    
+    @acr_preparer()
+    def test_upload_oci_manifest_stream(self, containerregistry_endpoint):
+        repo = self.get_resource_name("repo")
+        manifest = self.create_oci_manifest()
+        manifest_bytes = json.dumps(manifest.serialize()).encode("utf-8")
+        manifest_stream = BytesIO(manifest_bytes)
+        
+        client = self.create_registry_client(containerregistry_endpoint)
+        
+        client.upload_manifest(repo, manifest_stream)
```

Comment:
How can we validate that this test succeeded?
Is there a `get` call that we can do to validate that the manifest uploaded successfully?

Reply:
Have added download_manifest() call which include get_manifest() after.